tests/unit/__snapshots__/storybook.spec.ts.snap

```diff
@@ -82,6 +82,12 @@ exports[`Storyshots Core/Run Another Flow Block Styled Default 1`] = `
 </div>
 `;
 
+exports[`Storyshots Core/Set Contact Property Clear Action 1`] = `
```

Comment:
Obviously not ideal to have the empty snapshot. Is this something you can fix? I know we still have it for other sidebars.

Reply:
I agree with you, this is something we need to fix later, maybe after July delivery